tests/functional/pointerevent_capture_suppressing_mouse-manual.js

```diff
@@ -0,0 +1,30 @@
+define(function(require) {
+	var registerSuite = require('intern!object');
+	var w3cTest = require('../support/w3cTest');
+	var name = 'pointerevent_capture_suppressing_mouse-manual';
+
+	registerSuite({
+		name: name,
+
+		main: function() {
+			return w3cTest(this.remote, name + '.html')
+				.findById('target0')
+					.moveMouseTo(50, 50)
+					.clickMouseButton(0)
+					.end()
+				.findById('target1')
+					.moveMouseTo(50, 30)
+					.clickMouseButton(0)
+					.end()
+				.findById('btnCapture')
+					.moveMouseTo(50, 4)
+					.pressMouseButton(0)
+					.end()
+				.findByTagName('body')
```

Comment:
This is missing three steps:

* Put your mouse over the purple rectangle and then move it out. Nothing should happen
* Put your mouse over the black rectangle. pointerover and pointerenter should be logged inside of it.
* Move your mouse out of the black rectangle. pointerout and pointerleave should be logged inside of it